### PR TITLE
Single card

### PR DESCRIPTION
**Pull request by balkamile**
a card for one person

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+				/>
+				<span class="person-name">Maksim Jaroslacevas</span>
+			</div>
+			<div class="last-time-seen">
+				<div class="icon">
+					<svg
+						xmlns="http://www.w3.org/2000/svg"
+						class="ionicon"
+						viewBox="0 0 512 512"
+					>
```
Comment (AlexShukel) [inline example/index.html]: Use " alt=<...> />" instead of inlined `svg` tag. Same for all icons.

File: example/index.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+			<div class="title">
+				<img
+					class="profile-photo"
+					src="./assets/Default_pfp.svg.png"
+					alt="Profile photo"
+				/>
+				<span class="person-name">Maksim Jaroslacevas</span>
+			</div>
+			<div class="last-time-seen">
+				<div class="icon">
```
Comment (AlexShukel) [inline example/index.html]: There is no need for separate `div` wrapper, as you can pass `class` directly to `svg` or `img`. Same with all icons.

**Review by Maksasj — COMMENTED** [review]
looks good.

**Review by AlexShukel — COMMENTED** [review]
Ok

**Pull request merged**